pkg/configuration/base/resources/proxy.go

```diff
@@ -14,9 +14,9 @@ import (
 
 const (
 	openshiftGlobalProxy = "cluster"
-	envHTTPProxyName     = "HTTP_PROXY"
-	envHTTPSProxyName    = "HTTPS_PROXY"
-	envNoProxyName       = "NO_PROXY"
+	envHTTPProxyName     = "http_proxy"
```

Comment:
Why not keeping both variables: lower case and upper case?
it doesn't cost anything.
And also, we probably need a way to forcibly disable them.